src/layouts/Layout.astro

```diff
@@ -14,35 +14,102 @@ const currentYear = Temporal.Now.plainDateISO().year;
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
-    <meta name="description" content="Astro description" />
+    <meta name="description" content="All Bugs Are Fixed - A Developer Blog" />
```

Comment:
"A developer blog"

Sentence case

All Bugs Are Fixed is the full proper title, everything else should be sentence case.

Update the AGENTS.md.to also explain this